conftest.py

```diff
@@ -240,11 +241,8 @@ def check_logs_for_errors(dtest_setup):
                     error_str = error.strip()
 
                 if error_str:
-                    logger.error("Unexpected error in {node_name} log, error: \n{error}"
-                                 .format(node_name=node.name, error=error_str))
```

Comment:
Was this just creating unnecessary logging noise before?

Reply:
it was the only thing showing the node_name, so it was needed to know where the logs came from